reddit_liveupdate/media_embeds.py

```diff
@@ -163,24 +163,18 @@ def __init__(self, url):
         self.allowed_oembed_types = {"video", "rich", "link", "photo"}
         self.oembed_params["key"] = g.embedly_api_key
 
+    def fetch_media(self):
```

Comment:
Do you need to make this a noop? You're already overriding the only caller `scrape()`.